### DISCUSSION TITLE
Welcome to pytorch_tabular Discussions!

### DISCUSSION THREAD

**Discussion by manujosephv** [manujosephv/pytorch_tabular#44] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  To get started with controbuting to the project, head over to [contributing.md](https://github.com/manujosephv/pytorch_tabular/blob/6e9e6e180523a634ba671b3f90fc80cc1206587f/docs/contributing.md)

**Comment by Jayant-kernel** [comment]
Hi @manujosephv! I'm Jayant.

I’m a big fan of PyTorch Tabular and would love to contribute as part of ESoC 2026. I noticed this repository is much more manageable than other "flooded" Batch 1 organizations.

Could you confirm if the project will be officially participating in ESoC 2026, perhaps in the upcoming Batch 2?

Thanks
